packages/terra-consumer-nav/src/Nav.scss

```diff
@@ -42,15 +42,14 @@
     user-select: none;
     width: var(--terra-consumer-nav-width, 320px);
     z-index: 2;
-
     @media screen and (max-width: $terra-tiny-breakpoint) {
       width: 100vw;
     }
 
     @media screen and (max-width: $terra-medium-breakpoint) {
       overflow: auto;
       padding-top: var(--terra-consumer--mobile-nav-padding-top, 0);
-      position: absolute;
+      position: relative;
```

Comment:
can you move it outside of the media query. It should apply to both desktop and mobile

Reply:
yes, moved. Thanks for the input.